src/qt/forms/createwalletdialog.ui

```diff
@@ -107,6 +107,9 @@
         <property name="text">
          <string>Descriptor Wallet</string>
         </property>
+        <property name="checked">
```

Comment:
maybe switch this around and have an unchecked "legacy wallet" option here instead?

Reply:
I think that may be a little confusing?

Reply:
I think there's something to be said for both options, but I think the current name is slightly better. It highlights what is new instead of what is old.

Reply:
I suppose you could have a tooltip text that explains in more detail, that there are "legacy wallets" and "descriptor wallets" and what is the difference from a user point of view.